lib/Runner.js

```diff
@@ -4,19 +4,26 @@ const AWS = require('aws-sdk');
 const EventEmitter = require('events');
 const BPromise = require('bluebird');
 const percentile = require('percentile');
+const https = require('https');
 
 const REPORT_REGEX = /REPORT RequestId: ([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\s+Duration: ([0-9.]+) ms\s+Billed Duration: ([0-9]+) ms\s+Memory Size: (\d+) MB\s+Max Memory Used: (\d+) MB\s+$/; // eslint-disable-line max-len
 const PERCENTILES = [50, 66, 75, 80, 90, 95, 98, 99];
 
 class Runner extends EventEmitter {
-  constructor(options = {}) {
+  constructor(options = { maxSockets: 2048 }) {
```

Comment:
Can you define MAX_SOCKETS constant and export it as a part of Runner class (`Runner.MAX_SOCKETS`)? This way you'd be able to use it as a default value for the `max-sockets` option.